infrastructure/lib/infrastructure-stack.ts

```diff
@@ -19,6 +19,7 @@ import { OpenSearchWAF } from "./stacks/waf";
 import { OpenSearchMetricsNginxCognito } from "./constructs/opensearchNginxProxyCognito";
 import { OpenSearchMetricsMonitoringStack } from "./stacks/monitoringDashboard";
 import { OpenSearchMetricsSecretsStack } from "./stacks/secrets";
+import {GitHubAutomationApp} from "./stacks/gitHubAutomationApp";
```

Comment:
NIT: Use the Intellij reformat code command on the added files to match the style

Reply:
Updated @bshien please check.